Web/src/components/games/PandaSequence/GameDisplay.tsx

```diff
@@ -35,6 +37,20 @@ const GameContainer = styled.div`
   text-shadow: 3px 3px 0px rgba(0, 0, 0, 0.1);
 `;
 
+const WobbleAnimation = keyframes`${wobble}`;
+
+const DisplayBalloon = styled(BalloonSVG)`
+  width: 70px;
+  margin: 12px;
+  animation: ${(tags: {duration: number}) => (tags.duration === 0
+    ? undefined : css`${tags.duration / 1000}s ${WobbleAnimation} ease-in-out infinite;`)}
+`;
+
+const InputBalloon = styled(BalloonSVG)`
+  width: ${(tags: {isselected: number}) => (tags.isselected === 1 ? '100px' : '70px')}
```

Comment:
Same thing `props` vs `tags`

Also, shouldn't it be `isSelected`?

Reply:
Ye I wanted to do that but I was getting an error:
```
Warning: React does not recognize the `isSelected` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `isselected` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
```
Come to think of it, `selected` does the job too and does not appear as jarring.